src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,45 @@
-export const PeopleFilters = () => {
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+const centuries = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || null;
+  const centuryParams = searchParams.getAll('centuries') || [];
```

Comment:
let's make a code little bit cleaner 
```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
  
  const query = searchParams.get('query') || '';
  const sex = searchParams.get('sex') || null;
  const centuryParams = searchParams.getAll('centuries') || [];
```